### PR TITLE
Support Python 3

### PR DESCRIPTION
**Pull request by di**

### CODE REVIEW

File: schematics_extensions/test/test_immutable_model.py [hunk 1]
```diff
@@ -30,6 +30,6 @@ class ImmutabilityErrorTest(unittest.TestCase):
     def test_error_messasge(self):
         error = ImmutabilityError(ModelForTesting(), 'the_key', 'the_value')
 
-        self.assertRegexpMatches(error.message, r"'the_value'")
-        self.assertRegexpMatches(error.message, r"'the_key'")
-        self.assertRegexpMatches(error.message, r"ModelForTesting")
+        self.assertRegexpMatches(error.args[0], r"'the_value'")
```
Comment (csojinb-picwell) [inline schematics_extensions/test/test_immutable_model.py]: They took away error.message?? That seems like a loss.

**Comment by csojinb-picwell** [comment]
+1

File: schematics_extensions/test/test_immutable_model.py (hunk 1 above)
Comment (di) [inline schematics_extensions/test/test_immutable_model.py]: Indeed: https://www.python.org/dev/peps/pep-0352/

Comment (di) [inline schematics_extensions/test/test_immutable_model.py]: (It's actually been deprecated since 2.6)

**Pull request merged**